docs/releases/saros-e_16.0.0.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Saros for Eclipse 16.0.0 Release Notes
+---
+
+:tada: We are happy to announce **Saros for Eclipse 16.0.0** :tada:
+
+The new Saros for Eclipse 16.0.0 contains multiple [bug fixes](#bug-fixes)
+and [improvements](#improvements).
+
+## Installation
+Saros for Eclipse 16.0.0 requires
+ - `JDK 8` or newer
+ - Eclipse 4.6 (Neon) or newer
+
+## Compatibility
+Saros for Eclipse 16.0.0 is not compatible with other Saros versions.
+
+## Feedback
+If you find any issue with Saros, please report it in our [issue tracker](https://github.com/saros-project/saros/issues).
+In case you are not sure whether you found a bug or simply want to talk about Saros, ask us at [Gitter](https://gitter.im/saros-project/saros/user).
+
+## Changes
+
+Beside multiple internal reworks, [bug fixes](#bug-fixes) and [improvements](#improvements) we also
+[removed our Whiteboard](https://www.saros-project.org/contribute/deprecated/whiteboard.html).
+It was to brittle and does not fit into our plan to support further Editors/IDEs in the future.
+
+### Improvements
+* The `.git` directory is now ignored by default during a session ([#922](https://github.com/saros-project/saros/issues/922)).
+* Add Contacts Context Menu Item is enabled for Offline Contacts ([#19](https://github.com/saros-project/saros/issues/19))
```

Comment:
This entry and the next entry need adjusting as well in my opinion. (The initial review comment was addressing the range from line 30 to line 32. This might not have been apparent.)